optimization/physical_constants.py

```diff
@@ -0,0 +1,44 @@
+"""Physical Constants Used for 1D simulation
+*** All values are bulk flow values averaged axially across the core ***
+"""
+import math
+
+def fuel_cond(T):
+    """Estimate CERMET fuel conductivity based on T. Use a correlation from Webb
+    and Charit "Analytical Determination of thermal conductivity of W-UO2 and
+    W-UN CERMET nuclear fuels. Correlation provided for 60% UN
+    """
+
+    kc = 1.841e-19*math.pow(T,6) - 2.097e-15*math.pow(T,5) +\
+    9.721e-12*math.pow(T,4) - 2.369e-8*math.pow(T,3) +\
+    3.283e-5*math.pow(T,2) - 0.0267*T + 63.18
```

Comment:
indent

Reply:
PEP8: "Following the tradition from mathematics usually results in more readable code:"
-> math sign at the beginning of the line rather than at the end of the previous.